src/views/Shipments.vue

```diff
@@ -154,4 +175,12 @@ export default defineComponent({
     }
   }
 })
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+@media (min-width: 991px) {
+  ion-header > div {
+    display: flex;
```

Comment:
@amansinghbais why not use the same pattern as job manager?

Reply:
@dt2patel sir, The same pattern was used in the pipeline page of Job Manager app. Took reference from there only.